### PR TITLE
Design page layout

### PR DESCRIPTION
**Pull request by HaiTranVar**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,7 +2,7 @@ source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby "2.4.6"
-
+gem 'bootstrap-sass', '3.3.7'
```
Comment (ducminhvu) [inline Gemfile]: `'` -> `"`

File: app/assets/javascripts/staticpages.coffee [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (ducminhvu) [inline app/assets/javascripts/staticpages.coffee]: remove this file

File: app/assets/stylesheets/staticpages.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Staticpages controller here.
```
Comment (ducminhvu) [inline app/assets/stylesheets/staticpages.scss]: remove this file

File: app/helpers/application_helper.rb [hunk 4]
```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title(page_title = '')
```
Comment (ducminhvu) [inline app/helpers/application_helper.rb]: remove ()
`'` -> `"`

File: app/helpers/application_helper.rb [hunk 5]
```diff
@@ -1,2 +1,10 @@
...
+    base_title = "Vareal Viet Nam"
```
Comment (ducminhvu) [inline app/helpers/application_helper.rb]: i18n

File: app/helpers/staticpages_helper.rb [hunk 6]
```diff
@@ -0,0 +1,2 @@
+module StaticpagesHelper
```
Comment (ducminhvu) [inline app/helpers/staticpages_helper.rb]: remove this file

File: app/views/layouts/_footer.html.erb [hunk 7]
```diff
@@ -0,0 +1,12 @@
+<footer class="footer">
+    <p>Room No. 22, 8th Floor, DETECH TOWER II Building, 107 Nguyen Phong Sac</p>
```
Comment (ducminhvu) [inline app/views/layouts/_footer.html.erb]: i18n

File: app/views/layouts/_header.html.erb [hunk 8]
```diff
@@ -0,0 +1,13 @@
+
```
Comment (ducminhvu) [inline app/views/layouts/_header.html.erb]: remove this empty line

File: app/views/layouts/application.html.erb [hunk 9]
```diff
@@ -1,15 +1,18 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>OnlineExam</title>
+    <title><%= full_title(yield(:title)) %></title>
```
Comment (ducminhvu) [inline app/views/layouts/application.html.erb]: remove ()
`full_title yield(:title)`

File: app/views/layouts/application.html.erb [hunk 10]
```diff
@@ -1,15 +1,18 @@
...
     <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
-
-    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag    'application', media: 'all',
```
Comment (ducminhvu) [inline app/views/layouts/application.html.erb]: 1 space

File: app/controllers/staticpages_controller.rb [hunk 11]
```diff
@@ -0,0 +1,11 @@
+class StaticpagesController < ApplicationController
+  def home
+  end
+
+  def help
+  end
+
+  def about
+    I18n.locale = :en
```
Comment (mai-123) [inline app/controllers/staticpages_controller.rb]: this is unnecesary

File: app/controllers/staticpages_controller.rb [hunk 12]
```diff
@@ -0,0 +1,4 @@
+class StaticpagesController < ApplicationController
```
Comment (ducminhvu) [inline app/controllers/staticpages_controller.rb]: `StaticPagesController`

File: app/views/layouts/_header.html.erb [hunk 13]
```diff
@@ -0,0 +1,11 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to t(".logo"),'https://www.vareal.vn/vn/', id: "logo" %>
```
Comment (ducminhvu) [inline app/views/layouts/_header.html.erb]: link to root path

File: app/views/layouts/application.html.erb [hunk 14]
```diff
@@ -1,15 +1,18 @@
...
-    <title>OnlineExam</title>
+    <title><%= full_title yield(:title) %></title>
     <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
-
-    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag 'application', media: 'all',
+                                              'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= render 'layouts/shim' %>
```
Comment (ducminhvu) [inline app/views/layouts/application.html.erb]: `'` -> `"`

File: config/locales/views/en.yml [hunk 15]
```diff
@@ -0,0 +1,22 @@
...
+      logo: "OnlineExam"
+      log_in: "LOG IN"
+      help: "HELP"
+      home: "HOME"
+  staticpages:
+    about:
+      title1: "ABOUT COMPANY"
+    home:
+      title2: "HOME"
+    help:
```
Comment (ducminhvu) [inline config/locales/views/en.yml]: remove this

File: config/routes.rb [hunk 16]
```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
+  root 'staticpages#home'
```
Comment (ducminhvu) [inline config/routes.rb]: ' > "

File: app/views/staticpages/home.html.erb [hunk 17]
```diff
@@ -0,0 +1,3 @@
+<% provide(:title, "Home")%>
```
Comment (ducminhvu) [inline app/views/staticpages/home.html.erb]: i18n

File: app/views/static_pages/home.html.erb [hunk 18]
```diff
@@ -0,0 +1,3 @@
+<% provide(:title,t('.label'))%>
```
Comment (ducminhvu) [inline app/views/static_pages/home.html.erb]: remove ()
' > "
space before ,

File: config/locales/views/en.yml [hunk 19]
```diff
@@ -0,0 +1,17 @@
+en:
+  layouts:
+    footer:
+      home: "HOME"
+      about: "ABOUT US"
```
Comment (ducminhvu) [inline config/locales/views/en.yml]: remove this

File: config/locales/views/en.yml [hunk 20]
```diff
@@ -0,0 +1,17 @@
...
+      contact: "CONTACT US"
+      address1: "Room No. 22, 8th Floor, DETECH TOWER II Building, 107 Nguyen Phong Sac"
+      address2: "Dich Vong Hau Ward, Cau Giay District, Hanoi 123114 "
+      address3: "VIET NAM"
+    header:
+      logo: "OnlineExam"
+      log_in: "LOG IN"
+      help: "HELP"
```
Comment (ducminhvu) [inline config/locales/views/en.yml]: remove this

File: app/views/layouts/_footer.html.erb [hunk 21]
```diff
@@ -0,0 +1,5 @@
+<footer class="footer">
+    <p><%= t(".address1") %></p>
+    <p><%= t(".address2")%></p>
```
Comment (ducminhvu) [inline app/views/layouts/_footer.html.erb]: space after last %

File: app/views/layouts/_header.html.erb [hunk 22]
```diff
@@ -0,0 +1,11 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to t(".logo"), root_path, id: "logo" %>
+    <nav>
+      <ul>
+        <li><%= link_to t(".log_in"),'#' %></li>
```
Comment (ducminhvu) [inline app/views/layouts/_header.html.erb]: space after ,

File: app/views/static_pages/home.html.erb [hunk 23]
```diff
@@ -0,0 +1,3 @@
+<% provide(:title, t(".label"))%>
+<h1><%= t ".label" %></h1>
```
Comment (ducminhvu) [inline app/views/static_pages/home.html.erb]: `label` -> `title`

File: app/views/static_pages/home.html.erb [hunk 24]
```diff
@@ -0,0 +1,3 @@
...
+<p>Find me in app/views/staticpages/home.html.erb</p>
```
Comment (ducminhvu) [inline app/views/static_pages/home.html.erb]: remove this

File: app/views/layouts/_header.html.erb [hunk 25]
```diff
@@ -0,0 +1,11 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to t(".logo"), root_path, id: "logo" %>
+    <nav>
+      <ul>
+        <li><%= link_to t(".log_in"),"#" %></li>
```
Comment (ducminhvu) [inline app/views/layouts/_header.html.erb]: space after ,

File: app/views/static_pages/home.html.erb [hunk 26]
```diff
@@ -0,0 +1,2 @@
+<% provide(:title, t(".title"))%>
```
Comment (ducminhvu) [inline app/views/static_pages/home.html.erb]: remove ()

File: app/assets/stylesheets/custom.scss [hunk 27]
```diff
@@ -0,0 +1,56 @@
...
+  margin-top: 55px;
+  padding-top: 5px;
+  border-top: 1px solid #eaeaea;
+  color: white;
+}
+
+/* header  */
+
+header.navbar.navbar-fixed-top.navbar-inverse{
+background: #13547a;
```
Comment (ducminhvu) [inline app/assets/stylesheets/custom.scss]: ident

File: app/assets/stylesheets/custom.scss [hunk 28]
```diff
@@ -0,0 +1,56 @@
...
+  background: #13547a;
+  margin-top: 55px;
+  padding-top: 5px;
+  border-top: 1px solid #eaeaea;
+  color: white;
+}
+
+/* header  */
+
+header.navbar.navbar-fixed-top.navbar-inverse{
```
Comment (ducminhvu) [inline app/assets/stylesheets/custom.scss]: space before {

File: app/assets/stylesheets/custom.scss [hunk 29]
```diff
@@ -0,0 +1,56 @@
...
+background: #13547a;
+}
+
+header ul li{
```
Comment (ducminhvu) [inline app/assets/stylesheets/custom.scss]: space before {

File: app/assets/stylesheets/custom.scss [hunk 30]
```diff
@@ -0,0 +1,56 @@
...
+  float: right;
+  color: white;
+  display: inline-block;
+}
+
+header ul a{
```
Comment (ducminhvu) [inline app/assets/stylesheets/custom.scss]: space before {

**Comment by ducminhvu** [comment]
OK

**Pull request merged**